gen/typedef.go

```diff
@@ -64,6 +64,14 @@ func typedef(g Generator, spec *compile.TypedefSpec) error {
 
 		<$v := newVar "v">
 		<$x := newVar "x">
+
+		<- if isPrimitiveType .>
+		// <typeName .>Ptr returns a pointer to a <$typedefType>
+		func (<$v> <typeName .>) Ptr() *<$typedefType>{
+			return &<$v>
```

Comment:
A test for this in quick_test would be nice.

You want to add another case [here](https://github.com/thriftrw/thriftrw-go/blob/ff6c015d46769061e36ceaa685adfe68b2c1afde/gen/quick_test.go#L514) for when the kind is `thriftTypedef`, and the object has a `Ptr` method, to do something similar to [testEnumPtr](https://github.com/thriftrw/thriftrw-go/blob/ff6c015d46769061e36ceaa685adfe68b2c1afde/gen/quick_test.go#L777).

This will make sure that the Ptr methods on all generated types get exercised.